tools/nni_cmd/url_utils.py

```diff
@@ -29,6 +29,8 @@
 
 CLUSTER_METADATA_API = '/experiment/cluster-metadata'
 
+TUNER_DATA_API = '/experiment/tuning-data'
```

Comment:
Maybe Tuning...

Reply:
Keep consistent with other url

Reply:
It's for the consistent,
In this file, 
```
EXPERIMENT_API = '/experiment'

CLUSTER_METADATA_API = '/experiment/cluster-metadata'

TUNER_DATA_API = '/experiment/tuning-data'

CHECK_STATUS_API = '/check-status'

TRIAL_JOBS_API = '/trial-jobs'

TENSORBOARD_API = '/tensorboard'
```
all naming methods point to "[uppercase]_API = '/[lowercase]'"

**So why you use TUNER_DATA correspond to tuning-data?**

Reply:
You should make your words more clearly instead of just throwing out a short cut. `Maybe Tuning...` can be more easily understood as changing the url behind. `Tuning` will be firstly matched to the `tuning` in the url. Besides, if your point is to change the variable name, it should be `TUNING`, not `Tuning`. But anyway I will rename it.

Reply:
ok